spec/lib/api/api_config_spec.rb

```diff
@@ -22,11 +22,21 @@
 
     describe 'identifiers' do
       let(:api_feature_identifiers) do
-        feature_identifiers { |set, id| set.add(id) }
-      end
+        collection_settings.each_with_object(Set.new) do |(_key, config), ids|
+          action_configs  = config.to_h.select { |k, v| k.to_s.end_with?("_actions") }.values
+          action_configs += Array(config.resource_entities).map { |e| Array(e.entity_actions) }
+
+          action_configs.each do |action_config|
+            action_config.each do |_verb, actions|
+              actions.each do |action|
+                ids.merge(Array(action[:identifier]))
+                ids.merge(Array(action[:identifiers]).flat_map { |i| Array(i[:identifier]) })
```

Comment:
This effectively fixes the bug in the test...it never checked :identifiers (plural) before.